src/sentry/static/sentry/app/components/events/eventEntries.tsx

```diff
@@ -54,16 +56,41 @@ export const INTERFACES = {
   spans: SpansInterface,
 };
 
-class EventEntries extends React.Component {
+const defaultProps = {
+  isShare: false,
+  showExampleCommit: false,
+  showTagSummary: true,
+};
+
+// Custom shape because shared view doesn't get id.
+type SharedViewOrganization = {
+  slug: string;
+  id?: string;
+  features?: Array<string>;
+};
+
+type Props = {
+  // This is definitely required because this component would crash if
+  // organization were undefined.
+  organization: SharedViewOrganization;
+
+  event: Event;
+  group?: Group;
```

Comment:
maybe we could move these optional props to the final of the object?

Reply:
done